### PR TITLE
Feat/external keyboard 01

### PR DESCRIPTION
**Pull request by cmgustavo**
Use physical keyboard to enter amount (delete and enter for continue).

Waiting for: https://github.com/kevinejohn/react-native-keyevent/pull/78

### CODE REVIEW

**Comment by gabrielbazan7** [comment]
https://github.com/cmgustavo/bitpay-app/pull/1

**Comment by gabrielbazan7** [comment]
It would be nice to have this functionality in the PIN feature.

File: src/Root.tsx [hunk 1]
```diff
@@ -123,6 +123,7 @@ import BillStack, {
   BillStackParamList,
 } from './navigation/tabs/shop/bill/BillStack';
 import InAppNotification from './components/modal/in-app-notification/InAppNotification';
+import KeyEvent from 'react-native-keyevent';
```
[the remark this answers is not in the document]
Comment (cmgustavo) [reply src/Root.tsx:126]: nop, thanks!

**Comment by cmgustavo** [comment]
Removed in favor of https://github.com/bitpay/bitpay-app/pull/876

**Pull request closed without merging**